public/index.html

```diff
@@ -5,79 +5,108 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
 
+    <link rel="stylesheet" href="./css/fluent.css">
+    <link rel="stylesheet" href="./css/font-awesome.min.css">
     <link href="https://fonts.googleapis.com/css?family=Sarina" rel="stylesheet">
-    <link rel="stylesheet" href="style.css">
 
     <title>Project Fluent</title>
   </head>
   <body>
-    <div class="main">
-      <header class="header">
-        <h1 class="header__wordmark">
-          fluent
+    <div>
+      <nav id="topbar">
+        <div class="logo">
+          <a href="/">fluent</a>
+        </div>
+        <ul>
+          <li>
+            <a href="#">Home</a>
+          </li>
+          <li>
+            <a href="/fluent/guide">Documentation</a>
+          </li>
+          <li>
+            <a href="/play">Online Editor</a>
+          </li>
+          <li>
+            <a href="https://github.com/projectfluent/fluent/wiki">Wiki</a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+    <div>
+      <header id="intro">
+        <h1>
+          Fluent
         </h1>
-        <nav class="header__nav">
-          <a class="link" href="http://projectfluent.io/fluent/guide">Syntax Guide</a>
-          <a class="link" href="http://projectfluent.io/play">Online Editor</a>
-          <a class="link" href="https://github.com/projectfluent">GitHub</a>
-          <a class="link" href="https://groups.google.com/forum/#!forum/mozilla.tools.l10n">Mailing list</a>
-          <a class="link" href="https://github.com/projectfluent/fluent/wiki">Wiki</a>
-        </nav>
+        <h2>A localization paradigm designed to unleash the expressive power of the natural language.</h2>
+        <a href="https://github.com/projectfluent/fluent/wiki/Integrating-Fluent" class="cta">Get Started</a>
```

Comment:
I remember you mentioned you wanted to write the _Fluent Guide for Developers_ and link it here instead of the wiki page. Can you file an issue about it please?